CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 
 ### Bugfixes
 
+- [#6042](https://github.com/influxdata/influxdb/issues/6042): CreateDatabase failure on Windows, regression from v0.11.0 RC
```

Comment:
The changelog should probably go in v0.11.0 since it's a regression in the release candidate and we haven't released v0.11.0. If we don't get this change into the official v0.11.0, then it can get moved to this section.